.github/scripts/docker_push_script.sh

```diff
@@ -13,12 +13,13 @@ fi
 
 # Push those images
 for image in "${images[@]}"; do
-
     if [[ ${GITHUB_REF} == 'refs/heads/master' ]]; then
         docker push ${IMAGE_REPO}/${image}:testing
-
         docker tag ${IMAGE_REPO}/${image}:testing ${GITHUB_CONTAINER_REGISTRY}/${image}:testing
         docker push ${GITHUB_CONTAINER_REGISTRY}/${image}:testing
+    else
+        # append the branch name to testing tags for when we're building older versions of anax for testing
+        docker push ${IMAGE_REPO}/${image}:testing_${GH_BRANCH}
```

Comment:
The Docker build command in the Makefile produces the image with the `testing` and `<version>` tag. I think you need to tag it with `testing_${GH_BRANCH}` before pushing.

`docker tag ${IMAGE_REPO}/${image}:testing ${IMAGE_REPO}/${image}:testing_${GH_BRANCH}`